scrapy/utils/gz.py

```diff
@@ -51,8 +51,12 @@ def gunzip(data):
     return output
 
 _is_gzipped_re = re.compile(br'^application/(x-)?gzip\b', re.I)
+_is_octetstream_re = re.compile(br'^(application|binary)/octet-stream\b', re.I)
```

Comment:
Doing `is not None` explicitly makes it a bool though. If we drop the `is not None` then we're going to return None or True instead of True or False. I doubt it makes a difference though. 

```python
return _is_gzipped(ctype) or
            (_is_octetstream(ctype) and
             cenc in (b'gzip', b'x-gzip')))
```

If we want to be really pedantic we could probably also drop the parantheses too.